pkgs/development/python-modules/duct-py/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+# tests
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "duct-py";
+  version = "0.6.4";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "oconnor663";
+    repo = "duct.py";
+    rev = version;
+    hash = "sha256-4ja/SQ9R/SbKlf3NqKxLi+Fl/4JI0Fl/zG9EmTZjWZc=";
+  };
+
+  pythonImportsCheck = [ "duct" ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  disabledTests = [
+    "test_full_env"
+  ];
```

Explain why this test was disabled: "requires networking", "writes to $HOME"

```suggestion
  disabledTests = [
    # <reasons why>
    "test_full_env"
  ];
```